bpf/process/types/basic.h

```diff
@@ -854,6 +854,21 @@ FUNC_INLINE void copy_reverse(__u8 *dest, uint len, __u8 *src, uint offset)
 	}
 }
 
+#ifdef __LARGE_BPF_PROG
+FUNC_INLINE void file_copy_reverse(__u8 *dest, uint len, __u8 *src, uint offset)
```

Comment:
we could maybe add a comment on the fact that it's almost a copy of:
https://github.com/cilium/tetragon/blob/c638b45a5e1e9467b4505ec3adb9dbf4a26823fa/bpf/process/types/basic.h#L833
but with a different PATH_MASK and why it's needed so that we don't get confused later on.

Reply:
Thinking about this (and accepting that I said we could do with a new function), as it's FUNC_INLINE we could probably get away with passing the mask as a parameter. So both copy_reverse and file_copy_reverse could just be calls to another FUNC_INLINE _copy_reverse which has the relevant mask as a parameter. WDYT?